lib/options.py

```diff
@@ -228,6 +228,21 @@ def __init__(self):
                     Default: false.
                     """))
 
+        parser.add_argument(
+                "--luatestdir",
+                dest="luatestdir",
+                default="../third_party/luatest",
```

Comment:
Can we search it using `LUA_PATH` by default?

I mean, we don't need the `bin/luatest` wrapper (it just makes things more complicated for us in the case), we can just do `tarantool -e 'require('luatest.cli_entrypoint')()'`.

Reply:
I think, we don't ever need this `--luatestdir` argument or env variable. Just correctly configured lua paths.

Reply:
The thing is I don't want to setup `LUA_PATH` when running tests in Tarantool with `test-run.py`. I just want everything to work by default, without any extra parameters or environment variables. Moreover, I'd like `test-run.py --env` to setup `LUA_PATH` so that I can run tests with `luatest`.

Reply:
In other words, yes, we could require developers to setup `luatest` by themselves for running Tarantool tests, but (a) this would be inconvenient and (b) could result in non-obvious errors caused by using an old version of `luatest`. Better checkout `luatest` as a submodule in Tarantool and make `test-run` use it automatically IMO.

Reply:
Done in https://github.com/tarantool/test-run/pull/455.